weaviate/collections/classes/config.py

```diff
@@ -1187,6 +1187,11 @@ class _NestedProperty:
     nested_properties: Optional[List["NestedProperty"]]
```

Comment:
Does this field need to be mapped to `out["nestedProperties"]` also? Same goes for `index_filterable` and `index_searchable` in `_NestedProperty`

Reply:
done